src/signature.c

```diff
@@ -62,17 +62,11 @@ gboolean signature_init(GError **error)
 
 	ret = OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG | OPENSSL_INIT_LOAD_CRYPTO_STRINGS, NULL);
 	if (!ret) {
-		unsigned long err;
-		const gchar *data;
-		int flags;
-
-		err = ERR_get_error_line_data(NULL, NULL, &data, &flags);
 		g_set_error(
 				error,
 				R_SIGNATURE_ERROR,
 				R_SIGNATURE_ERROR_CRYPTOINIT_FAILED,
-				"Failed to initialize OpenSSL crypto: %s",
-				(flags & ERR_TXT_STRING) ? data : ERR_error_string(err, NULL));
```

Comment:
In some cases, data can contain relevant information. We should add a helper to use as `g_autofree gchar *err_str = r_get_openssl_error_string();`.